descriptions/0/api.intercom.io.yaml

```diff
@@ -1492,6 +1492,296 @@ paths:
+              examples:
+                Unauthorized:
+                  value:
+                    type: error.list
+                    request_id: b68959ea-6328-4f70-83cb-e7913dba1542
+                    errors:
+                    - code: rate_limit_exceeded
+                      message: Exceeded rate limit of 5 pending reporting dataset export jobs
+              schema:
+                "$ref": "#/components/schemas/error"
+  "/export/reporting_data/{job_identifier}":
+    get:
+      summary: Get export job status
+      tags: [Export]
+      parameters:
+        - name: Intercom-Version
+          in: header
+          schema:
+            "$ref": "#/components/schemas/intercom_version"
+        - name: app_id
+          in: query
+          description: The Intercom defined code of the workspace the company is associated
+            to.
+          required: true
+          schema:
+            type: string
+        - name: client_id
+          in: query
+          required: true
+          schema:
+            type: string
+        - name: job_identifier
+          description: Unique identifier of the job.
+          in: query
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: Job status returned successfully
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  job_identifier:
+                    type: string
+                    example: 'job1'
+                  status:
+                    type: string
+                    example: 'complete'
+                  download_url:
+                    type: string
+                  download_expires_at:
+                    type: string
+        '404':
+          description: When job not found
+          content:
+            application/json:
+              examples:
+                Not found:
+                  value:
+                    type: error.list
+                    request_id: b68959ea-6328-4f70-83cb-e7913dba1542
+                    errors:
+                    - code: not_found
+                      message: "Export job not found for identifier: job1"
+              schema:
+                "$ref": "#/components/schemas/error"
+        '504':
+          description: When job has failed
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  job_identifier:
+                    type: string
+                    example: 'job1'
+                  status:
+                    type: string
+                    example: 'failed'
+                  download_url:
+                    type: string
+                    example: ''
+                  download_expires_at:
+                    type: string
+                    example: ''
+  "/export/reporting_data/get_datasets":
+    get:
```

Comment:
Usually wouldn't have get in the path as well as being the HTTP method.

Reply:
But if it matches the behaviour we can update the docs for now. While it's in Unstable though we could change to align with other API endpoints.

Reply:
Yes I think that would be clearer and more consistent with the rest of the APIs